semgrep/semgrep/metric_manager.py

```diff
@@ -112,7 +130,7 @@ def send(self) -> None:
 
             try:
                 r = requests.post(
-                    METRICS_ENDPOINT, json=metrics, timeout=10, headers=headers
+                    METRICS_ENDPOINT, json=metrics, timeout=2, headers=headers
```

Comment:
Could also use `version.VERSION_CHECK_TIMEOUT` here, although that will conflate two things.